settings.py

```diff
@@ -34,10 +31,10 @@
 # If this is changed back to locmem, make sure to use it from "caching" (by
 # default in Django for locmem, a timeout of "0" means "don't cache it", while
 # on other backends it means "cache forever"):
-#      'BACKEND': 'caching.backends.locmem.LocMemCache'
+#      'BACKEND': 'django.core.cache.backends.locmem.LocMemCache'
 CACHES = {
     'default': {
-        'BACKEND': 'caching.backends.memcached.MemcachedCache',
+        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
```

Comment:
I thiiiiink this could also land upfront, the relevant memcached changes in regards to how timeouts work landed in previous Django versions already. Needs some checking though.